index.js

```diff
@@ -93,6 +94,10 @@ function buildResponseMessage(prDetails) {
     }
     if (!isValidPullRequestBody(prDetails.body)) {
         message += '1. PR Description\n';
+        if (process.env.ENABLE_KEYWORD_CHECKER != undefined
+            && process.env.ENABLE_KEYWORD_CHECKER.toLowerCase() === "true") {
```

Comment:
So the JSON would be part of the repo? If it is, wouldn't updating the forks reset their settings?

I'm planning to have variables only for features that make the bot less general (e.g. checking for specific conventions is one).